app/src/main/res/layout/activity_main.xml

```diff
@@ -5,6 +5,8 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:layout_gravity="left"
```

Comment:
<sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: Use "start" instead of "left" to ensure correct behavior in right-to-left locales [RtlHardcoded]